_includes/post-categories.html

```diff
@@ -0,0 +1,10 @@
+<span class="post-categories">{%- if page.categories.size == 1 -%}Category{%- else -%}Categories{%- endif -%}:&nbsp;
+  {%- for category in page.categories -%}
+    {%- if site.minima.archive_page.categories -%}
+      <a href="{{ site.minima.archive_page.categories }}/{{ category | slugify }}/">{{ category }}</a>
```

Comment:
The intention here isn't clear.
What exactly is `site.minima.archive_page.categories` supposed to be? Is it just an arbitrary prefix in the URL?

Additionally, this doesn't respect the user's `baseurl` setting.

Reply:
`site.minima.archive_page.categories` is a path to the categories page. 

Yes, it is arbitrary, but I should add the `baseurl`.